libdnf/repo/DependencySplitter.cpp

```diff
@@ -25,7 +25,7 @@
 namespace libdnf {
 
 static const Regex RELDEP_REGEX = 
-    Regex("^(\\S*)\\s*(<=|>=|<|>|=)?\\s*(.*)$", REG_EXTENDED);
+    Regex("^(\\S*)\\s*(<=|>=|<|>|=)?\\s*([^ \t]*)$", REG_EXTENDED);
```

Comment:
Alternatively: 
```Regex("^(\\S*)\\s*(<=|>=|<|>|=)?\\s*(\\S*)$", REG_EXTENDED);```

Reply:
I would prefer the latter alternative - with \S*. It is used the same way in the beginning of this regex.
I suppose there is no way how to achieve having a provide containing whitespace character.